lib/role.rb

```diff
@@ -50,6 +52,14 @@ module Role
     end
   end
 
+  def billing_administrator?
```

Comment:
What does separating this out buy us? We've doing a validation in the model. I suppose it would make sure that it always returns false if the feature is off, even if someone manually messes with the database or there's a user who had this role before we implemented the change.